### PR TITLE
Feat/component/header: Fix McK logo aspect ratio and add navigate function in header

### PR DESCRIPTION
**Pull request by Cosmoboi-i**

### CODE REVIEW

**Comment by Cosmoboi-i** [comment]
image
image
image

**Review by Chetan-MckinseyBEBR — APPROVED** [review]

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          src="./assets/McK_Logo.png"
+          className="h-[70px] w-[155px] aspect-auto self-end pl-0 lg:mr-[180px] sm:mr-[90px] pt-0 pb-[5px]"
+        />
+        {hasNav && (
+          <div className="grid grid-cols-3 gap-10 items-end">
+            <button
+              className={
+                location === PROJECTS_ROUTE
+                  ? 'text-cyan border-b-[5px] border-b-cyan border-solid pt-0 pb-2.5 px-[5px]'
+                  : 'text-[white] hover:text-cyan hover:border-b-[5px] hover:border-b-cyan hover:border-solid pt-0 pb-[15px] hover:pb-2.5 px-[5px]'
```
Comment (Chetan-MckinseyBEBR) [inline src/components/Header/index.jsx:21]: have these class string in a variable and reuse them rather than repeating it

**Review by Chetan-MckinseyBEBR — CHANGES_REQUESTED** [review]
Added few comments

File: src/components/Header/index.jsx (hunk 1 above)
Comment (Cosmoboi-i) [reply src/components/Header/index.jsx]: Done

**Review by Chetan-MckinseyBEBR — APPROVED** [review]

**Pull request merged**